client/history/client.go

```diff
@@ -39,10 +39,10 @@ type clientImpl struct {
 	resolver        membership.ServiceResolver
 	tokenSerializer common.TaskTokenSerializer
 	numberOfShards  int
-	// TODO: consider refactor thriftCache into a separate struct
-	thriftCacheLock sync.RWMutex
-	thriftCache     map[string]historyserviceclient.Interface
-	rpcFactory      common.RPCFactory
+	// TODO: consider refactor InquireCache into a separate struct
+	InquireCacheLock sync.RWMutex
```

Comment:
why this change?

Reply:
This is a mistake when I refactor the naming. I will change it back. Thanks for reminding.